src/main/java/frc/robot/OperatorInterface.java

```diff
@@ -63,19 +63,10 @@ public OperatorInterface(final SubsystemManager subMan) {
         operatorPanelManager.addButton(RobotConstants.BUTTONS.SELECT_PRESET_2)
                 .whenPressed(commandFactory.hoodTrenchPreset())
                 .add();
-                
-        drive = subsystemManager.getDriveSubsystem();
-        
-        joystickManager.addButton(RobotConstants.BUTTONS.SNAP_TO_TARGET)
-                .whenPressed(commandFactory.snapToVisionTargetCommand().withTimeout(5))
-                .add();
-        
-        joystickManager.addButton(12)
-                .whenPressed(commandFactory.hoodHomeCommand())
-                .add();
         
-        joystickManager.addButton(RobotConstants.BUTTONS.DRIVER_SHOOT)
-                .whileHeld(commandFactory.fireCommand())
+        operatorPanelManager.addButton(RobotConstants.BUTTONS.FIRE)
```

Comment:
is there a reason we're not stopping the elevator after the firecommand? IE, if we always want to stop elevator after fire, we should stop the elevator as a part of fire